cstar/tests/integration_tests/blueprints/fixtures.py

```diff
@@ -5,7 +5,9 @@
 
 
 @pytest.fixture
-def modify_template_blueprint(tmpdir) -> Callable[[Path | str, dict[str, str]], Path]:
+def modify_template_blueprint(
+    tmpdir,
```

Comment:
consider adding your typehint
```suggestion
    tmpdir: pathlib.Path,
```